contracts/safeguard/sentinel/Sentinel.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity 0.8.17;
+
+import "./GuardedPauser.sol";
+import "./GuardedGovernor.sol";
+
+contract Sentinel is GuardedPauser, GuardedGovernor {
+    constructor(
```

Comment:
I don't think the constructor is necessary.

Reply:
This is for normal (non-upgradable) deployments

Reply:
Ack